spikes/lk_motion_kit/main.cpp

```diff
@@ -0,0 +1,152 @@
+using namespace leka;
+
+namespace {
+
+namespace motors {
+
+	namespace left {
+
+		namespace internal {
+
+			auto dir_1 = mbed::DigitalOut {MOTOR_LEFT_DIRECTION_1};
+			auto dir_2 = mbed::DigitalOut {MOTOR_LEFT_DIRECTION_2};
+			auto speed = CorePwm {MOTOR_LEFT_PWM};
+
+		}	// namespace internal
+
+		auto motor = CoreMotor {internal::dir_1, internal::dir_2, internal::speed};
+
+	}	// namespace left
+
+	namespace right {
+
+		namespace internal {
+
+			auto dir_1 = mbed::DigitalOut {MOTOR_RIGHT_DIRECTION_1};
+			auto dir_2 = mbed::DigitalOut {MOTOR_RIGHT_DIRECTION_2};
+			auto speed = CorePwm {MOTOR_RIGHT_PWM};
+
+		}	// namespace internal
+
+		auto motor = CoreMotor {internal::dir_1, internal::dir_2, internal::speed};
+
+	}	// namespace right
+
+}	// namespace motors
+
+namespace imu {
+
+	namespace internal {
+
+		CoreI2C i2c(PinName::SENSOR_IMU_TH_I2C_SDA, PinName::SENSOR_IMU_TH_I2C_SCL);
+		EventLoopKit event_loop {};
+
+	}	// namespace internal
+
+	CoreLSM6DSOX lsm6dsox(internal::i2c);
+	CoreAccelerometer accel(lsm6dsox);
+	CoreGyroscope gyro(lsm6dsox);
+
+}	// namespace imu
+
+IMUKit imu_kit(imu::internal::event_loop, imu::accel, imu::gyro);
+
+namespace motion::internal {
+
+	EventLoopKit event_loop {};
+
+}	// namespace motion::internal
+
+MotionKit motion_kit(motors::left::motor, motors::right::motor, imu_kit, motion::internal::event_loop);
+
+namespace rfid {
+
+	auto serial = CoreBufferedSerial(RFID_UART_TX, RFID_UART_RX, 57600);
+	auto reader = CoreRFIDReaderCR95HF(serial);
+
+}	// namespace rfid
+
+auto rfidkit = RFIDKit(rfid::reader);
+
+}	// namespace
+
+void onMagicCardAvailable(const MagicCard &card)
+{
+	switch (card.getId()) {
+		case (MagicCard::number_1.getId()):
+			motion_kit.rotate(1, Rotation::counterClockwise);
+			break;
+		case (MagicCard::number_2.getId()):
+			motion_kit.rotate(2, Rotation::clockwise);
+			break;
+		case (MagicCard::number_3.getId()):
+			motion_kit.rotate(3, Rotation::counterClockwise);
+			break;
+		case (MagicCard::number_4.getId()):
+			motion_kit.rotate(4, Rotation::clockwise);
+			break;
+		case (MagicCard::number_5.getId()):
+			motion_kit.rotate(5, Rotation::counterClockwise);
+			break;
+		case (MagicCard::number_6.getId()):
+			motion_kit.rotate(6, Rotation::clockwise);
+			break;
+		case (MagicCard::number_7.getId()):
+			motion_kit.rotate(7, Rotation::counterClockwise);
+			break;
+		case (MagicCard::number_8.getId()):
+			motion_kit.startStabilisation();
+			rtos::ThisThread::sleep_for(10s);
+			motion_kit.stop();
+			break;
+		case (MagicCard::number_9.getId()):
+			motion_kit.startStabilisation();
+			break;
+		case (MagicCard::number_10.getId()):
+			motion_kit.stop();
+			break;
+	}
+}
+
+auto main() -> int
+{
+	logger::init();
+
+	HelloWorld hello;
+	hello.start();
+
+	imu::lsm6dsox.init();
+	imu_kit.init();
+	motion_kit.init();
+	rfidkit.init();
+
+	auto callback = [](const MagicCard &card) { onMagicCardAvailable(card); };
+
+	rfidkit.onTagActivated(callback);
```

Comment:
You can directly set it 

```suggestion
	rfidkit.onTagActivated(onMagicCardAvailable);
```